kvbc/src/st_reconfiguration_sm.cpp

```diff
@@ -78,15 +82,25 @@ bool StReconfigurationHandler::handlerStoredCommand(const std::string &key, uint
     auto strval = std::visit([](auto &&arg) { return arg.data; }, *res);
     T cmd;
     deserializeCmfMessage(cmd, strval);
-    return handle(cmd, seqNum, current_cp_num, blockid);
+    std::optional<bftEngine::Timestamp> timestamp = std::nullopt;
+    auto value = ro_storage_.get(kvbc::kConcordInternalCategoryId, std::string{kvbc::keyTypes::timestamp_key}, blockid);
+    if (value) {
+      const auto &data = std::get<categorization::VersionedValue>(*value).data;
+      concord::messages::Timestamp cmf_ts;
+      deserializeCmfMessage(cmf_ts, data);
+      timestamp.emplace(
+          bftEngine::Timestamp{bftEngine::ConsensusTime{cmf_ts.time_since_epoch}, cmf_ts.request_position});
```

Comment:
Nit: Can remove the `bftEngine::Timestamp` part as emplace() takes the arguments to call the constructor itself.

Reply:
It fails, probably because `Timestamp` I have not defined a ctor. If it is not critical, I'd leave it.